src/perl/Bio/EnsEMBL/Pipeline/PipeConfig/BRC4_genome_compare_conf.pm

```diff
@@ -156,7 +156,6 @@ sub pipeline_analyses {
     { -logic_name  => 'Fasta_DNA',
       -module      => 'Bio::EnsEMBL::Pipeline::Runnable::BRC4::DumpFastaDNA',
       -parameters => {
-        hash_key => "insdc_fasta_dna",
```

Comment:
Good spot, however I think we need to make this change in the first tag we created as the pipeline is already not functional in the current branch because some python scripts have been moved.

Reply:
Ah maybe this PR should be moved to be against the legacy branch instead?

Reply:
Maybe we will merge it to the main branch for now and the legacy branch can be changed in a separate PR